recipes/blacktex/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "blacktex" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/blacktex-{{ version }}.tar.gz
+  sha256: 1a0660ea8438ae7c640977a19b5b2c326a3555d0af2e32eb06e67758a4f0d40a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Since this is pure Python, better to make this noarch:
```suggestion
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```